docs/sts/assume-role.md

```diff
@@ -82,7 +91,7 @@ aws_secret_access_key = foo12345
 > NOTE: In the following commands `--role-arn` and `--role-session-name` are not meaningful for MinIO and can be set to any value satisfying the command line requirements.
 
 ```
-$ aws --profile foobar --endpoint-url http://localhost:9000 sts assume-role --role-arn arn:xxx:xxx:xxx:xxx --role-session-name anything
+$ aws --profile foobar --endpoint-url http://localhost:9000 sts assume-role --policy '{"Version":"2012-10-17","Statement":[{"Sid":"Stmt1","Effect":"Allow","Action":"s3:*","Resource":"*"}]}' --role-arn arn:xxx:xxx:xxx:xxxx --role-session-name anything 
```

Comment:
Resource needs to be in arn format - can you change  "Resource":"*" to  "Resource":"arn:aws:s3:::*" in this example?

Reply:
It doesn't need to be in ARN "*" is allowed @poornas